### ISSUE TITLE
Can we identify articles that can be excluded using the values in column E of the traffic_articles spreadsheet?

### ISSUE THREAD

**Issue by piafaustino** [SiliconValleyInsight/in-the-news#33]
In the spreadsheet of traffic news articles, we did previously indicate manually which articles can be excluded. Anything with a value in column E (exclude) can be excluded:
https://docs.google.com/spreadsheets/d/1OBAVimrHCXa-4xPkiM-C1k4lE5-2C5slDdXxfQ0VKKc/edit#gid=1057720013

We can also just manually set exclude = Yes for each of these articles via the survey web-app. But can this work be reduced if we pre-load the database with info we already put in the spreadsheet? That's about 229 articles.

So basically:
if column E is not blank, then exclude = yes

?

**Comment by piafaustino** [comment]
Looks good!
